libvast/include/vast/system/actors.hpp

```diff
@@ -298,7 +297,7 @@ using index_actor = typed_actor_fwd<
   // erased, returns the nil uuid. When keep_original_partition is no: does an
   // in-place pipeline keeping the old ids, and makes a new partition
   // preserving the old one(s).
-  caf::replies_to<atom::apply, pipeline_ptr, std::vector<uuid>,
+  caf::replies_to<atom::apply, pipeline_ptr, std::unordered_map<uuid, type>,
```

Comment:
It would be great if you could update the description, it looks like it has gone out of sync with earlier changes already.
It is not clear why `unordered_map<uuid, type>` is used here. Are we actually accessing specific values by id or would a `vector<pair<uuid, type>>` be more appropriate?

Reply:
I updated the description in https://github.com/tenzir/vast/pull/2771/commits/564ee29d906a6c51ab0000db4625809b3edc7f2f

Your question made me realize that `std::vector<partition_info>` already does the trick. I removed those `std::unordered_map`s in https://github.com/tenzir/vast/pull/2771/commits/2bc28382f19ba1a396dcb7463c2e41bf370264d7